packages/web3-providers-http/src/index.ts

```diff
@@ -0,0 +1,58 @@
+import axios, { AxiosInstance } from 'axios';
+import Web3ProviderBase from 'web3-providers-base';
+import { ProviderOptions } from 'web3-providers-base/types';
+
+import { HttpRpcOptions, HttpRpcResponse, IWeb3Provider } from '../types';
+
+export default class Web3ProvidersHttp extends Web3ProviderBase implements IWeb3Provider {
+  private _httpClient: AxiosInstance | undefined
```

Comment:
is null better in this case?

Reply:
Removed `undefined` all together, no reason this shouldn't be set - should error otherwise (think it was just a temp fix to make tsc happy)